autoload/http.vim

```diff
@@ -151,12 +151,26 @@ endfunction
 function! s:new_response_buffer(request_buffer, response) abort
     let l:request_buffer_name  = bufname(a:request_buffer)
     let l:buffer_name = fnamemodify(l:request_buffer_name, ":r") . '.response.' . localtime() . '.http'
-    if g:vim_http_split_vertically
-      execute 'vert new ' . l:buffer_name
-    else
-      execute 'new ' . l:buffer_name
-    end
+    if g:vim_http_tempbuffer
+      for win in range(1, winnr('$'))
+        if getwinvar(win, 'temp')
```

Comment:
is `w:temp` a builtin or a standard thing to do? I can't see anything obvious in help.

If not can you prefix the name to `w:http_vim_temp` or similar to ensure that we don't clobber some buffer also setting `w:temp` from another plugin.